spec/std/socket/udp_socket_spec.cr

```diff
@@ -52,6 +52,58 @@ describe UDPSocket do
       client.close
       server.close
     end
+
+    is_darwin = {{ flag?(:darwin) }}
+    if is_darwin && family == Socket::Family::INET6
+      # Darwin is failing to join IPv6 multicast groups on the continuous integration server.
+      # However this is known to work on macOS Mojave with Darwin 18.2.0.
+      pending "can join and transmit to multicast groups"
+    else
+      it "can join and transmit to multicast groups" do
+        udp = UDPSocket.new(family)
+        udp.bind(unspecified_address, 2000)
+
+        udp.multicast_loopback = false
+        udp.multicast_loopback?.should eq(false)
+        udp.multicast_loopback = true
+        udp.multicast_loopback?.should eq(true)
+
+        udp.multicast_hops = 4
+        udp.multicast_hops.should eq(4)
+        udp.multicast_hops = 1
+        udp.multicast_hops.should eq(1)
+
+        addr = case family
+               when Socket::Family::INET
+                 expect_raises(Socket::Error) { udp.multicast_interface 0 }
+                 udp.multicast_interface Socket::IPAddress.new(unspecified_address, 0)
+                 Socket::IPAddress.new("224.0.0.254", 2000)
+               when Socket::Family::INET6
+                 expect_raises(Socket::Error) { udp.multicast_interface(Socket::IPAddress.new(unspecified_address, 0)) }
+
+                 # Darwin has a bug that prevents selecting the "default" interface.
+                 # https://lists.apple.com/archives/darwin-kernel/2014/Mar/msg00012.html
+                 udp.multicast_interface(0) unless is_darwin
```

Comment:
You can omit this check. `is_darwin` can never be true here, because IPv6 on darwin is already skipped in line 57.